test/numbers.jl

```diff
@@ -2771,7 +2771,8 @@ let types = (Base.BitInteger_types..., BigInt, Bool,
     for S in types
         for op in (+, -)
             if S === Float16 # type instability here?
-                @test_broken @inferred Base.promote_op(op, S)
+                # broken, fixme then remove this branch
```

Comment:
why is test_throws wrong? it achieves the test_broken purpose of being noisy and loud and making you fix it if the test changes result. we don't have a report right now so the count of  broken tests isn't visible anywhere except via grep